app/app.js

```diff
@@ -97,7 +96,9 @@ app.get('/landing-gpay.html', (req, res) => {
  * An Article.
  * TODO(dvoytenko): remove "/examples/" path
  */
-app.get(['/((\\d+))', '/examples/sample-pub/((\\d+))'], (req, res) => {
+app.get(['/config/:configId/((\\d+))', '/((\\d+))',
+    '/examples/sample-pub/((\\d+))'], (req, res) => {
```

Comment:
I think it'd be fine to now remove `/examples/sample-pub` URLs - they were here only for transition.

Reply:
done